ansible/library/lldpctl_facts.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# This ansible module is for gathering lldp facts from SONiC device.
+# It takes two argument 
+# asic_instance_id :- Used to specify LLDP Instance in Multi-asic platforms
+# skip_interface_pattern_list:- Used to specify interface pattern list to be skip for gathering lldp facts.
+import re
+from ansible.module_utils.basic import AnsibleModule
+
+def gather_lldp(module, lldpctl_docker_cmd, skip_interface_pattern_list):
+    rc, output, err = module.run_command(lldpctl_docker_cmd)
+    if output:
+        output_dict = {}
+        current_dict = {}
+        lldp_entries = output.split("\n")
+        skip_interface_pattern_str = '(?:% s)' % '|'.join(skip_interface_pattern_list) if skip_interface_pattern_list else None
```

Comment:
Double-quotes for consistency:
```suggestion
        skip_interface_pattern_str = "(?:% s)" % '|'.join(skip_interface_pattern_list) if skip_interface_pattern_list else None
```